compiler/back_end/cpp/run_one_golden_test.py

```diff
@@ -34,12 +34,24 @@ def main(argv):
     emboss_compiler = argv[2]
     emb_file = argv[3]
     golden_file = argv[4]
-    include_dirs = argv[5:]
+
+    include_dirs = []
+    compiler_flags = []
+    for arg in argv[5:]:
+        if arg.startswith("--import-dir="):
```

Comment:
This isn't very robust but I don't think `import-dir` actually works as intended. I just filed #236 , maybe the flag should be removed since it isn't used in current tests.